src/pages/Liquidity/components/RemoveLiquidityBox/index.tsx

```diff
@@ -110,8 +111,13 @@ export const RemoveLiquidityBox = ({ totalSupplyCP }: { totalSupplyCP: BigNumber
     })
   }
 
-  const detectTxFee = (gasUsed: BigNumber) => {
-    return gasUsed.mul(2).div(3).mul(5 * 10 ** 9)
+  const detectTxFee = (gasUsed: BigNumber, gasPrice: BigNumber) => {
```

Comment:
so many duplicated functions of the same functionality, please find a shared place an put them there